server.js

```diff
@@ -1,14 +1,16 @@
-'use strict';
-
 const express = require('express');
 const app = express();
 const config = require('config');
 const utils = require('./utils');
+
 // Used for BUR
 const client = require('./sdk').client;
 
-const evaluatorRouter = require('./routes/evaluator');
-const adminRouter = require('./routes/admin');
+// Middlewares
+const authorization = require('./middleware/authorization').authorization;
```

Comment:
Super mega minor thing, but if that file will only have the authorization middleware, expose only that as default:
`module.exports = authorization;` 

then you don't need to add `.authorization` here.

Reply:
nice catch! ty